src/dmd/backend/gsroa.d

```diff
@@ -39,6 +39,9 @@ extern (C++):
 
 int REGSIZE();
 
+alias SliceSize = REGSIZE;  // slices are all register-sized
+enum MaxSlices = 2;         // max # of pieces we can slice an aggregate into
```

Comment:
This name is not following the standard D naming convention. Not sure if you have your own convention in the backend.

Reply:
It isn't terribly consistent in the back end. Nor the front end.

Reply:
Yes, but why not follow the D naming convention when adding new symbols? It will slightly improve the code base. It will take long time, but eventually all symbols can follow the D naming convention, if we want to. Every new symbol with the correct naming convention will take as slight further towards that goal.